src/web/.eslintrc.js

```diff
@@ -0,0 +1,165 @@
+module.exports = {
+  // env: {
+  //   browser: true,
+  //   commonjs: true,
+  //   es2021: true,
+  // },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2021,
+  },
+  extends: [
+    'airbnb',
+    'plugin:prettier/recommended',
+    'plugin:promise/recommended',
+    'plugin:jest-playwright/recommended',
+    'plugin:import/recommended',
+  ],
+  plugins: ['prettier', 'promise', 'jest', 'anti-trojan-source'],
+  settings: {
+    'import/resolver': {
+      node: {},
+    },
+    react: {
+      version: '17.0',
+    },
+  },
+  overrides: [
+    // TypeScript for Next.js
+    {
+      files: ['src/web/**/*.ts', 'src/web/**/*.tsx'],
```

Comment:
I think you need to remove this bit from the root `.eslintrc.js` file, and only define the `src/web` stuff here.

Reply:
Isn't it going through all the files with `*.ts` and `*.tsx` in `src/web`? I'm not sure what should be removed in here

Reply:
There is no `src/web` inside `src/web`, which is where you are right now.

The root `.eslintrc.js` needs to not also double-add this override (i.e., you are doing it here).